### PR TITLE
README: Update with group members

### PR DESCRIPTION
**Pull request by Corbin-Jarrett**
This commit adds group member names to the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Capstone
 2024-2025 Mechatronics Group 7 Capstone
+
+Members: Corbin Jarrett, Sabrina Sousa, Couper Smith, Julia Meeson, Magalie Durepos-Letourneau.
```
Comment (sousas8) [inline README.md:4]: Létourneau

Comment (sabrina-v-sousa) [inline README.md]: Létourneau

**Review by Sabrina-V-Sousa — CHANGES_REQUESTED** [review]
add é to Létourneau

File: README.md (hunk 1 above)
Comment (Corbin-Jarrett) [reply README.md]: Fixed

**Review by Sabrina-V-Sousa — APPROVED** [review]

**Pull request merged**